nuntium/views.py

```diff
@@ -214,3 +215,25 @@ def get_context_data(self, **kwargs):
         context = super(MessagesPerPersonView, self).get_context_data(**kwargs)
         context['person'] = self.person
         return context
+
+
+class MessagesFromPersonView(ListView):
+    model = Message
+    template_name = 'nuntium/message/from_person.html'
+
+    def dispatch(self, *args, **kwargs):
+        self.writeitinstance = get_object_or_404(WriteItInstance, slug=self.kwargs['slug'])
+        self.message = get_object_or_404(Message, slug=self.kwargs['message_slug'])
+        return super(MessagesFromPersonView, self).dispatch(*args, **kwargs)
+
+    def get_queryset(self):
+        qs = super(MessagesFromPersonView, self).get_queryset()
+        qs = qs.filter(writeitinstance=self.writeitinstance).\
+            filter(public=True).filter(confirmated=True).\
```

Comment:
Do we now have a single way of checking to see if something is confirmed, or are there still two different ways? I seem to remember there was both a `confirmated` column and the existence of a confirmation object to check.